src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -366,7 +366,7 @@ def check_if_imaging_fits_into_memory(imaging: ImagingExtractor) -> None:
 
 def _imaging_frames_to_hdmf_iterator(
     imaging: ImagingExtractor,
-    iterator_type: str = "v2",
+    iterator_type: Optional[str] = "v2",
```

Comment:
@h-mayorquin  `Optional` means that `None` can be a value. In this case, this arg can only take the values 'v1' and 'v2', so `Optional` is not appropriate. See documentation [here](https://docs.python.org/3/library/typing.html#typing.Optional).

Reply:
oh I see, you can take None as a value